web/modules/custom/hoeringsportal_public_meeting/hoeringsportal_public_meeting.module

```diff
@@ -12,7 +12,8 @@ use Drupal\node\NodeInterface;
  * Set last meeting time for public meeting, based on Pretix field time_to values.
  */
 function hoeringsportal_public_meeting_node_presave(NodeInterface $node) {
-  if ($node->bundle() == 'public_meeting') {
+  $user = Drupal::currentUser();
+  if ($user->id() > 0 && $node->bundle() == 'public_meeting') {
```

Comment:
I case you mean "user is authenticated” (which I suppose you do), using `Drupal::currentUser()->isAuthenticated()` will make the code easier to understand.

Use `===` for strict comparison.

Reply:
Fixed